<!-- -*- mode: XML -*- -->
<launch>

    <arg name="fixed_frame" default="map"/>
    <arg name="odom_frame" default="odom"/>    
    <arg name="use_odom" default="true"/>

    <node pkg="ugv_control" type="path_node" name="path_node" output="screen">
        <param name="fixed_frame" value="$(arg fixed_frame)"/>
        <param name="odom_frame" value="$(arg odom_frame)"/>        
        <param name="use_odom" value="$(arg use_odom)"/>
    </node>

</launch>
